src/js/stories/ListStories.jsx

```diff
@@ -95,26 +89,114 @@ function RendererOptions() {
 
 function KeyboardFocus() {
     const buttonStyle = { margin: 10, flexShrink: 0 };
+    const outer = {
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'space-around',
+        alignItems: 'center',
+        padding: 10,
+        height: 400,
+    };
 
     return (
-        <div style={{...style, height: 300}}>
-            <p>This Layout is useful for demonstrating keyboard accessibility and focus behavior.</p>
+        <div style={outer}>
```

Comment:
We haven't really been using classes in any of the storybooks but we could at some point if we wanted to make the design of them more consistent. Inline styles are sorta convenient with the webpack hot reloading...